src/serializer.ts

```diff
@@ -0,0 +1,91 @@
+import {
```

Comment:
I wonder if we should implement a formatter as a prettier plugin
Its a common use case for Chevrotain and it is also in fact a back to text serializer.

Reply:
Will look into how to write the plugin to prettier.

Reply:
it may also increase the scope of this quick-start too much, maybe it belong in a different example, I am not sure...

On the other hand its also provides similar functionality to a serializer...

Relevant references:
- https://github.com/bd82/toml-tools/tree/master/packages/prettier-plugin-toml
- https://github.com/prettier/plugin-xml/blob/master/src/print.js
- https://github.com/jhipster/prettier-java (this is a really complex plugin, may be less good as an example).